scripts/skk.sh

```diff
@@ -0,0 +1,34 @@
+set -e
+
+. ./common.sh skk $1
+
+# Use PkgConfig::LibSKK
+sed -i '' 's/find_package(LibSKK REQUIRED)/pkg_check_modules(LibSKK REQUIRED IMPORTED_TARGET "libskk" REQUIRED)/' CMakeLists.txt
+sed -i '' 's/LibSKK::LibSKK/PkgConfig::LibSKK/' src/CMakeLists.txt
+
+# Patch to support $XDG_DATA_DIR
+git checkout src/skk.cpp
+git apply ../patches/skk-XDG_DATA_DIR.patch
+
+# Download the default dict
+dict_dir=$DESTDIR$INSTALL_PREFIX/share/skk-dict
+mkdir -p $dict_dir
+cd $dict_dir
+wget -nc https://skk-dev.github.io/dict/SKK-JISYO.L.gz
+gunzip -f SKK-JISYO.L.gz
+mv SKK-JISYO.L $dict_dir/SKK-JISYO.L
+cd -
+
+# Build fcitx5-skk
+install_deps libxkbcommon pcre2 glib libgee libskk json-glib
+f5m_configure -DENABLE_QT=OFF
```

Comment:
There is an option `SKK_DEFAULT_PATH` defaulting to `/usr/share`, but it is merely a default path used in the Add Dict dialog, but the GUI config tool is not enabled by our build.

Ideally, the default path should be `~/Library/fcitx5/share/skk-dict/...`, but that requires code changes.